### PR TITLE
refactor(Tile and Tile Select): match Figma and utilize Tailwind

### PR DESCRIPTION
**Pull request by justinhough**
**Related Issue:** #2470 

## Summary

The alignment on calcite-tile-select for title and inputs was caused by the `calcite-checkbox` and `calcite-radio` using `inline-block` rather than `block`.

I checked the following components to verify that the change did not adversely affect anything else.

- calcite-radio-button
- calcite-radio-group
- calcite-tree

### CODE REVIEW

**Comment by justinhough** [comment]
@julio8a This PR won't pass since the tests for both `calcite-checkbox` and `calcite-radio` are checking against the element being `inline` in the build and the change needed for Tile Select need those two elements to be `block` in order to layout properly. Let me know how you want to proceed on this. Thanks.

**Comment by julio8a** [comment]
@driskull or @jcfranco since we need to change from inline to block to get the visual we need, should we update the tests?

**Comment by driskull** [comment]
Yeah update the tests 👍

**Review by bstifle — APPROVED** [review]
screener had some weird stuff but verified that they were fine.

lookin good!

**Review by julio8a — APPROVED** [review]
Nice! Thanks for updating the test!

**Comment by julio8a** [comment]
@justinhough, can you resolve conflicts in this PR?

**Comment by julio8a** [comment]
Screener is showing visual changes on popover, I don't think it fully loaded when it took the screenshot because it's on the top left.

**Comment by julio8a** [comment]
@jcfranco, in this case, should be approve screener and merge the PR? or merge without approving screener. 

My understanding is that once we approve it, the approved version (top left popover, which is incorrect) becomes the new norm, is this correct?

**Comment by driskull** [comment]
@julio8a I think it's better to `ignore` it here:

image

File: src/tests/commonTests.ts [hunk 1]
```diff
@@ -65,8 +65,8 @@ export async function renders(
   const element = await page.find(getTag(componentTagOrHTML));
 
   expect(element).toHaveAttribute(HYDRATED_ATTR);
-  expect(await element.isVisible()).toBe(options?.visible ?? true);
-  expect((await element.getComputedStyle()).display).toBe(options?.display ?? "inline");
+  expect(await element.isVisible()).toBe(options.visible ?? true);
```
Comment (driskull) [inline src/tests/commonTests.ts:68]: @julio8a these two lines should be reverted back. Can you revert both of these changes?

File: src/tests/commonTests.ts [hunk 2]
```diff
@@ -65,8 +65,8 @@ export async function renders(
...
+  expect((await element.getComputedStyle()).display).toBe(options.display ?? "block");
```
Comment (driskull) [inline src/tests/commonTests.ts:69]: fallback display needs to be "inline". The default display for a custom element is "inline". Please revert this.

File: src/tests/commonTests.ts (hunk 1 above)
Comment (driskull) [reply src/tests/commonTests.ts:68]: options are optional so the `?` needs to be there.

**Pull request merged**